### PR TITLE
Add frontpage for tracingstore + datastore

### LINKED ISSUE #4868: A friendly frontpage for the datastore
When accessing a datastore, users are greeted by this error message. Would it be much trouble to add a friendlier page that just says hi and maybe reports the health check?

Screenshot 2020-10-12 at 18 39 13

### PR DESCRIPTION
**Pull request by fm3**
Standalone Datastores and Tracingstores replied with Play’s default “action not found” page on `/` or `/index.html` requests. This PR adds a less denying view there.

### Steps to test:
- run standalone datastore `sbt "webknossosDatastore/run 9090 -Dconfig.file=webknossos-datastore/conf/standalone-datastore.conf"`, navigate to http://localhost:9090
- you should see this page:
image

### Issues:
- fixes #4868 

------
- [x] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

**Comment by normanrz** [comment]
The page has too much info on it for my taste. I'd strip it down to a big "Go to webKnossos" link with some deferred info in the bottom. I wouldn't link to build info or health check either. (I included the health check result in my mockup, but it is not necessary).

datastore-placeholder

**Comment by fm3** [comment]
Yeah, health check on that page is not so simple, unfortunately. New version looks like this:
image

File: webknossos-datastore/app/views/datastoreFrontpage.scala.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    a {color: black; text-decoration: none; font-size: 2em}
+    #hint {color: #777; margin-top: 4em}
+</style>
+</head>
+<body>
+<a href="@(wKUri)" id="wKlink">
+  <img src="https://static.webknossos.org/images/oxalis.svg" alt="" />
+  <span>Go to webKnossos</span>
+</a>
+<p id="hint">This is the webKnossos @(kind) “@(name)”.</p>
```
Comment (normanrz) [inline webknossos-datastore/app/views/datastoreFrontpage.scala.html:24]: Maybe use JS for the healthcheck?
```suggestion
<p id="hint">This is the webKnossos @(kind) “@(name)”. Healthcheck: <span id="healthcheck">Loading...</span></p>
<script>
fetch("/data/health")
  .then((r) => r.text())
  .then((status) => {
    document.getElementById("healthcheck").innerHTML = status;
  })
  .catch((err) => {
    document.getElementById("healthcheck").innerHTML = "Error";
  });
</script>
```

**Comment by normanrz** [comment]
The screen looks tasty now 😋

File: webknossos-datastore/app/views/datastoreFrontpage.scala.html (hunk 1 above)
Comment (fm3) [reply webknossos-datastore/app/views/datastoreFrontpage.scala.html]: yeah, good idea :)

**Review by normanrz — APPROVED** [review]
I am happy with this now. But @youri-k should check the actual backend code ;-)

**Review by youri-k — APPROVED** [review]
LGTM :shipit:

**Pull request merged**